src/main/java/athena/EditCommand.java

```diff
@@ -0,0 +1,36 @@
+package athena;
+
+import java.io.IOException;
+
+public class EditCommand extends Commands {
+    protected static int taskIndex;
+    protected static String taskName;
+    protected static String taskTime;
+    protected static String taskDuration;
+    protected static String taskDeadline;
+    protected static String taskRecurrence;
+    protected static String taskImportance;
+    protected static String taskNotes;
+
+    public EditCommand(int index, String name, String time, String duration, String deadline, String recurrence, String importance, String notes) {
```

Comment:
Consider renaming the "time" parameter to "startTime" here too as this is the time the task should task.